analysis/test/Batch/Batch.py

```diff
@@ -46,9 +44,9 @@
 
 for datasetName in RunFiles:
     fileList = datadir + datasetName + '.txt'
-    jobName = analysis+'_'+datasetName 
+    jobName = anaName+'_'+datasetName 
 
-    Dirname = "{}/src/nano/analysis/test/Batch/{}".format(os.environ['CMSSW_BASE'],jobName)
+    Dirname = "%s/src/nano/analysis/test/Batch/%s"%(cmsswBase,jobName)
```

Comment:
Can you just write cmsswBase like that? it doesn't seem to be connected to anything